ros2_odometry_node:
  ros__parameters:
    publish_tf: true
    odom_frame: "odom"
    base_frame: "base_link"
    drive_type: "ACKERMANN"